CHANGELOG.md

```diff
@@ -7,6 +7,51 @@ Mapbox welcomes participation and contributions from everyone.
+    * The following 4 functions have been removed: 
+        * `public func coordinate(for point: CGPoint, in view: UIView? = nil) -> CLLocationCoordinate2D`
+        * `public func point(for coordinate: CLLocationCoordinate2D, in view: UIView? = nil) -> CGPoint`
+        * `public func coordinateBounds(for view: UIView) -> CoordinateBounds`
+        * `public func rect(for coordinateBounds: CoordinateBounds, in view: UIView? = nil) -> CGRect`
+    * `OrnamentSupportableMapView` is not internal.
+
+- #### Ornaments
+    * `LayoutPosition` has been deprecated in favor of `OrnamentPosition`.
+    * `LayoutVisibility` has been depracted in favor of `OrnamentVisibility`.
+    * `showsLogoView` has been renamed to `_showsLogoView`.
+    * `showsCompass` and `showsScale` have been depracted. You should now use the visibility properties to set how the Compass and Scale Bar should be shown.
+
+- #### Foundation
+    * `cancelTransitions` has been renamed to `cancelAnimations`.
+    * [`setCamera()`](https://github.com/mapbox/mapbox-maps-ios/pull/250/files#diff-8fa667141ac423a208a6e7036ed759e7e52fc6940bd58834c1935c2c6ead9c65L177) with individual params,has been deprecated in favor of [this](https://github.com/mapbox/mapbox-maps-ios/blob/edbf08e37975c81c7ee1cbc4bb046c48d522d306/Sources/MapboxMaps/Foundation/Camera/CameraManager.swift#L140) one that requires `CameraOptions`.
+    * The following camera convenience functions have been deprecated:
+        * `public func transitionCoordinateBounds(newCoordinateBounds: CoordinateBounds, animated: Bool = false)`
+        * `public func transitionCoordinateBounds(to newCoordinateBounds: CoordinateBounds, edgePadding: UIEdgeInsets, animated: Bool = false, completion: ((UIViewAnimatingPosition) -> Void)? = nil)`
+        * `public func transitionVisibleCoordinates(newCoordinates: [CLLocationCoordinate2D], edgePadding: UIEdgeInsets, animated: Bool = false)`
+        * `public func transitionVisibleCoordinates(to newCoordinates: [CLLocationCoordinate2D], edgePadding: UIEdgeInsets, bearing: CLLocationDirection, duration: TimeInterval, animated: Bool = false, completion: ((UIViewAnimatingPosition) -> Void)? = nil)`
+        * `public func resetPosition()`
+        * `public func resetNorth(_ animated: Bool = false)`
+    * In `CameraAnimator`, `fractionComplete` is now of type `Double`, and `delayFactor` now returns a `Double`.
+    * `EventType` is internal.
+    * `MBXEdgeInsets` extension is internal.
+    * `ScreenCoordinate` extension is internal.
+    * `MapboxLogoView` has been renamed to `LogoView`.
+    * `MapboxLogoSize` has been renamed to `LogoSize`.
+
+- #### Style
+    * Initializer is now marked as internal.
+    * `styleUri` has been renamed to `uri`.
+    * The `url` property from `StyleURL` has been removed.
+
+- #### Expressions
+    * `ExpressionBuilder` has been renamed to `FunctionBuilder`.
```

Comment:
Why did we decide to do this?

Reply:
`ExpressionBuilder` seems more specific & clear to me

Reply:
I think this was decided when I left the meeting so not sure of the context.

Reply:
Okay, let's move forward with the change. I'll add a note to revisit this in the doc

Reply:
We undid this :)

Reply:
This was supposed to be the `@_functionBuilder` :)